--- aribb25-0.2.6/Makefile.am.orig	2014-10-21 21:08:42.000000000 +0200
+++ aribb25-0.2.6/Makefile.am	2015-02-15 21:34:12.561661026 +0100
@@ -6,7 +6,7 @@
         src/b_cas_card_error_code.h src/multi2_error_code.h src/td.c \
         src/ts_section_parser_error_code.h src/arib_std_b25.h src/b_cas_card.h \
         src/multi2.h src/ts_common_types.h src/ts_section_parser.h src/aribb25api.h
-libaribb25_la_LIBADD = $(AM_LIBS) $(pcsclite_CFLAGS)
+libaribb25_la_LIBADD = $(AM_LIBS) $(pcsclite_LIBS)
 libaribb25_la_CFLAGS = $(AM_CFLAGS) -Wall -Werror -fvisibility=hidden $(pcsclite_CFLAGS)
 
 pkginclude_HEADERS = src/arib_std_b25_error_code.h src/b_cas_card_error_code.h \
